src/styles.scss

```diff
@@ -1 +1,19 @@
-@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```

Comment:
This is already imported and saved under variable in `variables.scss` if you want to use it, use it from there. Remove this line.